eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```
@@ -261,24 +261,23 @@ extends:
                   WorkingDirectory: $(SdkRepoDirectory)
                   ScriptDirectory: $(SdkRepoDirectory)/eng/common/scripts
 
-              - ${{ if not(endsWith(variables['SdkRepoName'], '-pr')) }}:
-                - task: PowerShell@2
-                  displayName: Create pull request
-                  condition: and(succeeded(), eq(variables['HasChanges'], 'true'), ne(variables['Build.Reason'], 'PullRequest'))
-                  inputs:
-                    pwsh: true
-                    workingDirectory: $(SdkRepoDirectory)
-                    filePath: $(SdkRepoDirectory)/eng/common/scripts/Submit-PullRequest.ps1
-                    arguments: >
-                      -RepoOwner "$(SdkRepoOwner)"
-                      -RepoName "$(SdkRepoName)"
-                      -BaseBranch "main"
-                      -PROwner "azure-sdk"
-                      -PRBranch "$(PrBranch)-$(Build.BuildId)"
-                      -AuthToken "$(azuresdk-github-pat)"
-                      -PRTitle "$(PrTitle)-generated-from-$(Build.DefinitionName)-$(Build.BuildId)"
-                      -PRBody "$(GeneratedSDKInformation)"
-                      -OpenAsDraft $true
+              - task: PowerShell@2
+                displayName: Create pull request
+                condition: and(succeeded(), eq(variables['HasChanges'], 'true'), ne(variables['Build.Reason'], 'PullRequest'), not(endsWith(variables['SdkRepoName'], '-pr'))
```

Can you trigger a test run using private repo pipeline?
```suggestion
                condition: and(succeeded(), eq(variables['HasChanges'], 'true'), ne(variables['Build.Reason'], 'PullRequest'), not(endsWith(variables['SdkRepoName'], '-pr')))
```